stage1/init/common/pod.go

```diff
@@ -508,6 +522,11 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		unit.NewUnitOption("Service", "SyslogIdentifier", appName.String()),
 	}
 
+	if supportsNotify(p, appName.String()) {
+		opts = append(opts, unit.NewUnitOption("Service", "Type", "notify"))
+		opts = append(opts, unit.NewUnitOption("Service", "NotifyAccess", "all"))
```

Comment:
do you mind adding a small comment for the rationale of using `NotifyAccess=all`?